### DISCUSSION TITLE
使用PAGImageView播放，release打包比debug打包显得更卡

### DISCUSSION THREAD

**Discussion by pinixiong** [Tencent/libpag#2603] [Q&A]
机器环境：rk3568芯片，安卓11。
需要有多个 View 的存在，所以使用的是PAGImageView，pag文件就2kb。
排查过，不是混淆和压缩的问题，在debug编译里面minifyEnabled和shrinkResources配置true，也是不卡。
![配置](https://github.com/user-attachments/assets/461c38a9-682e-420d-93b2-8751827b938c)

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

如果你还在使用 libpag 且问题仍然存在，麻烦回复一下当前版本号（4.4 / 4.5 系列）和最小复现 demo，我们会优先跟进；如果不再需要跟进，本条就先关闭了。（stale-close）
